api/internal/handler/route/route.go

```diff
@@ -218,7 +220,11 @@ func (h *Handler) List(c droplet.Context) (interface{}, error) {
 			if input.Label != "" && !utils.LabelContains(obj.(*entity.Route).Labels, labelMap) {
 				return false
 			}
-
+			
+		        status, _ := strconv.Atoi(input.Status)
```

Comment:
Hi, I think this can be done before the `List` function. 
And we need to check the error. Return `400` if err is not nil.